cmd/peapod-to-fstree/main.go

```diff
@@ -46,7 +46,14 @@ func (s *onlyObjectsWithContainersStorage) Iterate(objHandler func(oid.Address,
 		}
 
 		return objHandler(addr, data, id)
-	}, nil)
+	}, func(addr oid.Address, err error) error {
+		_, found := s.containers[addr.Container()]
+		if !found {
+			log.Printf("skipping object without container '%s': %v", addr, err)
```

Comment:
```suggestion
			log.Printf("skipping corrupted object without container '%s': %v", addr, err)
```
we skip many-many objects, not all of them are logged